server/Makefile

```diff
@@ -126,3 +133,12 @@ docker-build:
 .PHONY: deploy
 deploy: kustomize yq
 	YQ="$(YQ)" KUSTOMIZE="$(KUSTOMIZE)" ./hack/deploy.sh "$(NAMESPACE)" "$(IMG)"
+
+.PHONY: package
+package: kustomize yq $(OUTDIR)
+	JWKS_URL="https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs" \
```

Comment:
I'd suggest to expose the JWKS_URL so we can easily overwrite it

Reply:
Fixed in adcd7a1.